public/New-DbaAvailabilityGroup.ps1

```diff
@@ -1,4 +1,4 @@
-function New-DbaAvailabilityGroup {
```

Comment:
I don't see a change here so there might be a problem with the encoding or the BOM. Please make sure that this line is not part of the diff.